department-of-veterans-affairs/caseflow
# Wmedders21/appeals 49036

Resolves [APPEALS-49036](https://jira.devops.va.gov/browse/APPEALS-49036)

# Description
Please explain the changes you made here.
- Adds `scheduled_in_timezone` to `ETL::HearingRecord#mirrored_hearing_attributes`
- Refactors `:distributed_to_specialty_case_team` string in `BVAAppealStatus` to fit within 32 char limit set by ETL schema for `appeals.status` - previously was 37 characters causing a failure during `ETLBuilderJob`

## Acceptance Criteria

## Testing Plan

1. Open rails console in sandbox mode in terminal `rails c --sandbox`
2. Destroy any seeded `AttorneyCaseReview` and `JudgeCaseReview` records with `nil` `appeal_id` values.
```
AttorneyCaseReview.where(appeal_id: nil).destroy_all
JudgeCaseReview.where(appeal_id: nil).destroy_all
```
3. Run the ETLBuilderJob `ETLBuilderJob.perform_now`
4. Inspect some `ETL::Hearing` records and verify that they have `scheduled_in_timezone` values
```
ETL::Hearing.limit(20).pluck(:scheduled_in_timezone)
```
5. Expect to see valid timezone strings returned
6. Exit rails console. This concludes the test



# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
